### PR TITLE
Russian Roulette Revamp

### PR DESCRIPTION
**Pull request by dapocalypse**
Russian Roulette has pretty much no risk. If you die you can just get cloned and come back. This means there is no consequence to playing. By gibbing the loser it ensures that a acual risk is present.

### CODE REVIEW

**Comment by FalseIncarnate** [comment]
I sense web editor...

I don't think gibbing the loser is the solution here though. You could make the revolver treat the loser as if they were a suicide (which technically it is) so they can't be cloned normally (there are ways around this but they are a little more involved than tossing them in the cloner directly) if your issue is that they can easily be revived. Gibbing is generally saved for special sources.

**Comment by dapocalypse** [comment]
Edit it now treats it as a suicide not gibs.

**Comment by Fox-McCloud** [comment]
What, no.

The Russian revolver is meant to be a casual bar-game, not a "lol takes you out of the round permanently" thing.

Saying there's no consequence to dying is a bit dishonest.

**Comment by dapocalypse** [comment]
Hmm?

**Comment by dapocalypse** [comment]
And how is it dishonest.

**Comment by dapocalypse** [comment]
I hardly understand how shooting yourself with a gun with the posibilkty of killing yourself is a 'casual bar game'

**Comment by tigercat2000** [comment]
I concur with Fox, this is kinda silly. The roulette game isn't supposed to be super high-stakes.

**Comment by dapocalypse** [comment]
(Sarcasm) Of course shooting yourself with the possibility of killing your self is low stakes.

**Comment by Tayyyyyyy** [comment]
Handing yourself over to medbay, possibly decapped, is the definition of high stakes.

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 1]
```diff
@@ -252,6 +252,7 @@
 
 /obj/item/weapon/gun/projectile/revolver/russian/proc/shoot_self(mob/living/carbon/human/user, affecting = "head")
 	user.apply_damage(300, BRUTE, affecting)
+	user.do_suicide(BRUTE, "Russian Revolver")
```
Comment (Tayyyyyyy) [inline code/modules/projectiles/guns/projectile/revolver.dm]: Did you test this? do_suicide doesn't set the mob's suiciding var to 1, so this doesn't prevent them from being cloned.

**Comment by Birdtalon** [comment]
A bit silly really and unnecessary.

**Comment by dapocalypse** [comment]
Please tell me that works

**Comment by Tayyyyyyy** [comment]
> Please tell me that works

Welll, you really should test your own PRs instead of depending on others to test them for you. But as it is, that doesn't even compile. The variable is `suiciding`, not `suicide`.

https://github.com/ParadiseSS13/Paradise/blob/92ae356a8c2bfb19409c23d6a7aea985c25fb40b/code/game/verbs/suicide.dm#L72-L75

**Comment by dapocalypse** [comment]
I just have to do this on my school ipad as i am at school. Therefore I cant test.

**Comment by dapocalypse** [comment]
Fixed

**Comment by Fox-McCloud** [comment]
This isn't meant to be something that takes you out of the round permanently; it's a game that heavily inconveniences you, if you lose (which it accomplishes).

This isn't something we want.

**Pull request closed without merging**